x-pack/plugins/ml/public/application/data_frame_analytics/pages/job_map/components/controls.tsx

```diff
@@ -57,9 +69,25 @@ function getListItems(details: object): EuiDescriptionListProps['listItems'] {
   });
 }
 
-export const Controls: FC<Props> = ({ analyticsId, modelId, details, getNodeData }) => {
+export const Controls: FC<Props> = ({
+  analyticsId,
+  details,
+  getNodeData,
+  modelId,
+  updateElements,
+}) => {
   const [showFlyout, setShowFlyout] = useState<boolean>(false);
   const [selectedNode, setSelectedNode] = useState<cytoscape.NodeSingular | undefined>();
+  const [isPopoverOpen, setPopover] = useState(false);
+
+  const canDeleteDataFrameAnalytics: boolean = checkPermission('canDeleteDataFrameAnalytics');
+  const deleteAction = useDeleteAction(canDeleteDataFrameAnalytics);
+  const { deleteItem, deleteTargetIndex, isModalVisible, openModal } = deleteAction;
+  const {
+    services: { notifications },
+  } = useMlKibana();
```

Comment:
We also have `useNotifications()` as a shortcut in `x-pack/plugins/ml/public/application/contexts/kibana/use_notifications_context.ts`.

Reply:
Updated in 626ab62bb02d68546b25cf3cbf527a0fd88cc1a9